### PR TITLE
Resolve JMS VM migration

### PR DESCRIPTION
**Pull request by chamil321**

### CODE REVIEW

File: modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/actions/jms/client/Send.java [hunk 1]
```diff
@@ -88,13 +89,25 @@ public BValue execute(Context context) {
 
         validateParams(bConnector);
 
+        BValue valueRef = new BBoolean(true);
+
+        if (context.isVMBasedExecutor()) {
+            context.getControlStackNew().currentFrame.returnValues[0] = valueRef;
+        } else {
```
Comment (shafreenAnfar) [inline modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/actions/jms/client/Send.java]: There is only one executor right ?

Comment (chamil321) [inline modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/actions/jms/client/Send.java]: yes, Updated PR after removing unwanted code

File: modules/ballerina-core/src/main/java/org/ballerinalang/bre/bvm/BLangVM.java [hunk 2]
```diff
@@ -942,7 +943,10 @@ private void exec() {
                         message = (BMessage) sf.refRegs[i];
                     }
                     context.setError(null);
-                    context.getBalCallback().done(message != null ? message.value() : null);
+                    // this should be fixed once JMS transactions are available as it requires callback
+                    if (((DefaultBalCallback) context.getBalCallback()).getParentCallback() != null) {
+                        context.getBalCallback().done(message != null ? message.value() : null);
```
Comment (shafreenAnfar) [inline modules/ballerina-core/src/main/java/org/ballerinalang/bre/bvm/BLangVM.java]: Talked to Hasitha and got to know that context.getBalCallback() could be null. We need to check that too.

File: modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/actions/jms/client/Send.java [hunk 3]
```diff
@@ -88,13 +89,21 @@ public BValue execute(Context context) {
...
+
         //Getting the map of properties.
-        BMap<BString, BString> properties = (BMap<BString, BString>) bConnector.getValue(0);
+        BMap<String, BString> properties = (BMap<String, BString>) bConnector.getValue(0);
 
         //Create property map to send to transport.
-        Map<String, String> propertyMap = properties.keySet()
-                .stream()
-                .collect(Collectors.toMap(BString::stringValue, k -> properties.get(k).stringValue()));
+//        Map<String, String> propertyMap = properties.keySet()
```
Comment (ruwanta) [inline modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/actions/jms/client/Send.java]: Better remove this commented out lines.

Comment (chamil321) [reply modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/actions/jms/client/Send.java]: I checked the file. It doesn't exist in ballerina anymore. 
Thanks

**Pull request merged**